.github/workflows/style-check.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
           pip install --upgrade clang-format
           which clang-format
           clang-format --version
-          git diff -U0 --no-color $(git merge-base origin/master HEAD) -- ':(exclude)*.js' |
+          git diff -U0 --no-color $(git merge-base origin/master HEAD) -- '*.cpp' '*.hpp' |
```

Comment:
This must also include .cpp.in and .hpp.in files. Generally I think the blacklist approach is better than the whitelist approach, so I'd much prefer if we could exclude the web UI from this instead.